cc_types.c

```diff
@@ -295,18 +295,36 @@ void create_enum(void)
 	maybe_bootstrap_error("enum statement");
 	struct type* head = calloc(1, sizeof(struct type));
 	require(NULL != head, "Exhausted memory while creating an enum\n");
+	struct type* i = calloc(1, sizeof(struct type));
+	require(NULL != i, "Exhausted memory while creating a struct indirection\n");
```

Comment:
Should this be `enum indirection`

Reply:
Fixed.

Reply:
Thanks. Rest is probably ok, but I want to test (a bit later today or tomorrow) with stage0-posix, to make sure we don't have any issues there.